.github/workflows/ci-checklist.yml

```
@@ -0,0 +1,38 @@
+
+      - name: Install DotNet Tools and Run Husky
+        run: |
+          dotnet tool restore && \
+          dotnet husky install && \
+          dotnet husky run -g "lint"
+
+      - name: Restore Dependencies
+        run: dotnet restore ${{ inputs.project-path }}
+
+      - name: Build Project
+        run: dotnet build --configuration Release --no-restore ${{ inputs.project-path }}
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)